keyboards/nuphy/air75_v2/ansi/ansi.c

```diff
@@ -0,0 +1,276 @@
+/*
+Copyright 2023 @ Nuphy <https://nuphy.com/>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "user_kb.h"
+#include "ansi.h"
+#include "usb_main.h"
+
+extern bool f_rf_sw_press;
+extern bool f_sleep_show;
+extern bool f_dev_reset_press;
+extern bool f_bat_num_show;
+extern bool f_rgb_test_press;
+extern bool f_bat_hold;
+extern uint16_t  no_act_time;
+extern uint8_t   rf_sw_temp;
+extern uint16_t  rf_sw_press_delay;
+extern uint16_t  rf_linking_time;
+extern user_config_t user_config;
```

Comment:
This really should be renamed to be keyboard related rather than user.  Eg, `kb_confg_t kb_config`.  That way it doesn't possibly collide with user configs.

Reply:
Thanks for your suggestion.

Reply:
> This really should be renamed to be keyboard related rather than user. Eg, `kb_confg_t kb_config`. That way it doesn't possibly collide with user configs.

Thanks for your suggestion.